### ISSUE TITLE
Select multiple folders in Reprioritize window

### ISSUE THREAD

**Issue by truth1ness** [hsoft/dupeguru#297]
How can I select multiple folders in the Reprioritize window to move to the right column in one chunk? I have many directories with tons of subfolders and I want to move the whole thing over in one chunk to the right for reprioritization but it won't let me select more than one at a time . Thanks.

**Comment by hsoft** [comment]
It's not possible at the moment. But I agree that in some cases (such as the one you mention), it would be a real time saver.

The Re-prioritize dialog is a relatively new feature and I'm not even sure that it's of much use to anyone except people with very specific needs. That is why this feature has many rough edges.

**Comment by truth1ness** [comment]
I think the feature is pretty awesome. When I was trying out other dedupe programs I kept wishing there was something like it. The select multiple would be great to have. 

Perhaps another option would be to show parent folders in the list. From what I can tell it only shows folders with files in it, so a top level folder that only has subfolders isn't displayed. Maybe those could be displayed and clicking on them simply highlights all the subfolders along with it. I think I still like the standard shift/command click abilities but this could be an additional way to select chunks more cleanly.

**Comment by hsoft** [comment]
For the record, a similar feature was asked in #282

**Comment by gordon-morehouse** [comment]
This bites me too on what I'd consider moderately complex deduping jobs.

**Comment by psychoadept** [comment]
If we could enter parent directories instead of each specific one, that would help.  I'm trying to compare files on computer to a backup, and there are over 1000 individual folders involved (17104 duplicates).

**Comment by vertigo220** [comment]
I agree the current design can be painfully slow. I just went through removing duplicates in a setup like this:

2008
  01
  02
  ...
  12
2009
  01
  02
  ...
  12
...
2020
  01
  02
  ...
Other folders

I wanted the images already sorted in the dated folders to be used as the references, and the images in the other folders to be considered duplicates. Unfortunately, this meant a painstaking process of clicking on the first folder in the left box, then clicking the arrow, then clicking the next folder, then the arrow, and so on, ad nauseam, a process made even worse by the fact I'm doing this remotely, so I have to deal with the lag that entails. If I could have _at least_ double-clicked the folders to send them to the other side, it would have been easier, but being able to shift- or ctrl-click to select multiple folders to add all at once would be ideal. Heck, even being able to select the one or two "other folders" and send them over and choose to set those as duplicate, vs automatically assigning the chosen folders to be reference folders, would have been substantially better. And as @truth1ness and @psychoadept mentioned, being able to select a parent would also be very helpful. If I could have selected an entry and had the option to customize it, then use wildcards, I could have, for example, selected 2017/03 and changed it to 2017/* and added that.

**Comment by msdobrescu** [comment]
The current implementation is absolutely counter-effective, to above I would add that at item prioritization adding to the right list, they should be removed from the left and vice-versa, i.e. they should move from one list to another, otherwise it is possible to add the same item several times - which is superfluous.

**Comment by Dobatymo** [comment]
@truth1ness @serg-z This can be closed now, can it?

**Comment by serg-z** [comment]
> @truth1ness @serg-z This can be closed now, can it?

@arsenetar

**Comment by arsenetar** [comment]
Per the original request yes, removing items from the list on one side when "moved" to the list on the other side is not in this though.
